nevergrad/benchmark/experiments.py

```diff
@@ -1023,7 +1023,7 @@ def control_problem(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     for sigma, func in zip(sigmas, funcs):
         f = func.copy()
         param: ng.p.Array = f.parametrization.copy()  # type: ignore
-        param.set_mutation(sigma=sigma).set_name(f"sigma={sigma}")
+        param[0][0].set_mutation(sigma=sigma).set_name(f"sigma={sigma}") # type: ignore
```

Comment:
@teytaud @LaurentMnr95 
Do we want to set the same sigma to each of the layers?
If so we need something like this (a bit hacky :s)
```suggestion
        for array in param[0]._content.values():
            array.set_mutation(sigma=sigma)
        param.set_name(f"sigma={sigma}")
```
@herilalaina the name of the mutation should be set on the full parametrization (param)
Also, since this is not an Array anymore, please update the typing (to ng.p.Instrumentation?) just above